File: src/main/java/com/example/demo/datastruct/FibonacciDemo.java
package com.example.demo.datastruct;

public class FibonacciDemo {
    public static void main(String[] args) {
        int[] a = new int[40];
        a[0] = 1;
        a[1] = 1;
        for (int i = 2; i < 40; i++) {
            a[i] = a[i - 1] + a[i - 2];
        }

        for (int i = 0; i < 40; i++) {
            System.out.print(a[i] + " ");
        }
    }
}

File: src/main/java/com/example/demo/datastruct/FibonacciDemo.kt
package com.example.demo.datastruct

fun main() {
    var a = IntArray(40)
    a[0] = 1
    a[1] = 1
    for (i in 2 until 40) {
        a[i] = a[i - 2] + a[i - 1]
    }
    for (tem in a) {
        print("$tem ")
    }
}